espnet/nets/pytorch_backend/e2e_tts_fastspeech.py

```diff
@@ -0,0 +1,686 @@
+        group.add_argument("--teacher-model", default=None, type=str, nargs="?",
+                           help="Teacher model file path")
+        parser.add_argument("--reduction-factor", default=1, type=int,
+                            help="Reduction factor")
+        # training related
+        group.add_argument("--transformer-init", type=str, default="pytorch",
+                           choices=["pytorch", "xavier_uniform", "xavier_normal",
+                                    "kaiming_uniform", "kaiming_normal"],
+                           help="How to initialize transformer parameters")
+        group.add_argument("--initial-encoder-alpha", type=float, default=1.0,
+                           help="Initial alpha value in encoder's ScaledPositionalEncoding")
+        group.add_argument("--initial-decoder-alpha", type=float, default=1.0,
+                           help="Initial alpha value in decoder's ScaledPositionalEncoding")
+        group.add_argument("--transformer-lr", default=1.0, type=float,
+                           help="Initial value of learning rate")
+        group.add_argument("--transformer-warmup-steps", default=4000, type=int,
+                           help="Optimizer warmup steps")
+        group.add_argument("--transformer-enc-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer encoder except for attention")
+        group.add_argument("--transformer-enc-positional-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer encoder positional encoding")
+        group.add_argument("--transformer-enc-attn-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer encoder self-attention")
+        group.add_argument("--transformer-dec-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer decoder except for attention and pos encoding")
+        group.add_argument("--transformer-dec-positional-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer decoder positional encoding")
+        group.add_argument("--transformer-dec-attn-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer decoder self-attention")
+        group.add_argument("--transformer-enc-dec-attn-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for transformer encoder-decoder attention")
+        group.add_argument("--duration-predictor-dropout-rate", default=0.1, type=float,
+                           help="Dropout rate for duration predictor")
+        group.add_argument("--init-encoder-from-teacher", default=True, type=strtobool,
+                           help="Whether to initialize encoder using teacher's parameters")
+        group.add_argument("--init-encoder-module", default="all", type=str,
```

Comment:
maybe `reset-encoder-module` is familiar to me because pytorch module always implements `m.reset_parameters()`

Reply:
fixed.